application/templates/data_provider_index.html

```
@@ -0,0 +1,88 @@
+    </div>
+  </div>
+
+  <form class="dl-filter-organisations-list__form filter-organisations-list__form--active govuk-!-margin-bottom-9" data-module="dl-list-filter-form">
+
+    <!-- Text filter -->
+    <label class="dl-list-filter__label govuk-label govuk-!-font-weight-bold" for="filter-providers-list">Find a data provider</label>
+    <input class="dl-list-filter__input govuk-input govuk-!-margin-bottom-3" type="text" id="filter-providers-list">
+
+  </form>
+
+  {% for group in provider_groups if group.providers|length > 0 %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)